templates/eu-packager-code-3-parts.156x90.tt.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="45px" height="26px" viewBox="0 0 255 152" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title></title>
```

Comment:
Maybe add/fill a title and description, since they could act as accessibility helpers.